Dockerfile

```diff
@@ -22,7 +23,7 @@ COPY overlay/ .
 RUN mkdir -p /opt/istio-releases && /usr/local/app/scripts/fetch_istio_releases.sh /opt/istio-releases
 RUN mkdir -p /var/cache/nginx
 
-RUN chown -R nginx:nginx /var/cache/nginx /etc/ssl /var/run /usr/share/nginx /usr/local/share/ca-certificates /usr/local/app/dashboards
+RUN chown -R nginx:nginx /var/cache/nginx /etc/ssl /var/run /usr/share/nginx /usr/lib/ca-certificates /usr/local/app/dashboards
```

Comment:
Was this tested with mirror mode on to ensure the nginx works properly ?

Reply:
@cbron Using `/usr/local/share/ca-certificates` the build was failing because it wasn't able to find `ca-certificates` inside /usr/local/share/. So I went into a SLE BCI image `(docker run -it registry.suse.com/suse/sle15:15.3)` to check where `ca-certificates` is and I found that inside `/usr/lib/` but not inside `/usr/local/share/`.
I also used the image `hackman91/istio-installer:v15.3` while installing istio on rancher and deployed demobookservice workloads and tested.

Reply:
@brendarearden after 2.6.3 calms down can you help them do a test with your mirror code to ensure nginx is spinning up correctly.

Reply:
https://github.com/rancher/istio-installer/blob/4731bc266fa041d7154cc44cd6b1ed3e56b3ddce/scripts/run.sh#L9 I think we missed this needing updated to match the new `/usr/lib/ca-certificates` location

Reply:
Sure, I will rebuild the image after adding this and test the image again.